src/utils/helper.ts

```diff
@@ -792,3 +792,9 @@ export function checkNested(obj, path: string) {
     }
     return true;
 }
+
+export function isTrue(t: any) {
+    return (
+        !isNullOrUndefined(t) && (t.toString().toLowerCase() === 'true' || typeof t === 'object')
```

Comment:
Does the function name IsTrue seem a little inappropriate for where the condition could be true when it's a object ?